src/by/naumenka/lesson21/Main.java

```diff
@@ -0,0 +1,56 @@
+package by.naumenka.lesson21;
+
+//Max,Winston,male,14
+//Jaime,Blossom,female,34
+//Darya,Naumenka,female,19
+
+import java.io.*;
+import java.util.*;
+
+public class Main {
+    public static void main(String[] args) throws IOException {
+        ArrayList<Person> people = new ArrayList<>();
```

Comment:
It's better to define variables more abstract, if we can.
Eg. `List<Person> people = new ArrayList<>();`